src/deploy/table_user_interest.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy maevsi:table_user_interest to pg
+
+BEGIN;
+
+CREATE TABLE maevsi.user_interest ( 
+    user_id uuid NOT NULL REFERENCES maevsi.account(id) ON DELETE CASCADE, 
```

Comment:
Let's call it `account_id` (it should be called `account_id` in all other places currently)